### ISSUE TITLE
The training data in the code is the same as the test data?

### ISSUE THREAD

**Issue by Anxia10** [danieltan07/dagmm#26]
The training data in the code is the same as the test data?

**Comment by chiachen-chang** [comment]
Yes, it's suitable to use the KDD dataset for unsupervised learning tasks. The training set includes only positive instances, while the testing set consists of both positive and negative ones. It's important to note that all the positive instances in the test set should be included in the training set, making this set-up appropriate for unsupervised learning.
